content/en/docs/tasks/security/proxy-cert/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Provision a certificate through a sidecar [Experimental]
```

Comment:
provisioning a certificate to your application through file

Reply:
Done.